LiskHQ/lisk
# Configuration of https timeouts being set in wrong place - Closes #2644

### What was the problem?

Configuration of https timeouts are configured regardless if ssl is enabled for the API

### How did I fix it?

Configuration of https timeouts are configured if ssl is enabled for the API

### How to test it?

Start the API and check for no error message with text: "Cannot set property 'headersTimeout' of undefined"

### Review checklist

* The PR resolves #2644
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
